[IMP] Point of Sale: Improvements on docs about discounts (#611)
odoo/documentation-user: point_of_sale/advanced_pricing_features/discounts.rst

.. image:: media/discounts_03.png
   :align: center
   :alt: View of the button to use for global discount via the pos interface

Click on it and, then, enter the wanted discount.

.. image:: media/discounts_04.png
   :align: center
   :alt: View of the discount offered in the payment summary

.. note::
   On this example, there is a global discount of 50% as well as a specific
   50% discount on oranges.

Apply time-limited discounts
============================

To activate time-limited discounts, you must activate the *Pricelists*
feature. To do so, go to :menuselection:`Point of Sales -->
Configuration --> Point of Sale` and open your PoS. Then, enable the
pricelist feature.

.. image:: media/discounts_05.png
   :align: center
   :alt: View of the pricelist feature

Once activated, you must choose the pricelists you want to make
available in the PoS and define a fault one.

Create a pricelist
------------------

By default, Odoo has a *Public Pricelist* configured. To create more,
go to :menuselection:`Point of Sale --> Products --> Pricelists`. Then
click on create.

When creating a pricelist, you can set several criteria to use a

> specific price: period, min. quantity, etc. You can also decide to apply

Author: I let you decide which version you prefer @jubodoo

that pricelist on specific products or on the whole range.

.. image:: media/discounts_06.png
   :align: center
   :alt: View of a time-limited pricelist for two products

Using a pricelist with the PoS interface
----------------------------------------

On the PoS interface, a new button appears. Use it to select a
pricelist.

.. image:: media/discounts_07.png
   :align: center
   :alt: View of the button to use for time-limited discounts via the pos interface

> If you click on it, the price is instantly updated to reflect the
> pricelist. Then, you can finalize the order.

Reviewer: ```suggestion
Now click on it to instantly updated the prices with the
pricelist selected. Then, you can finalize the order.
```
Something like this maybe. I just tried to "continue" the explanation considering the paragraph before.
